core/src/main/java/org/elasticsearch/action/support/ToXContentToBytes.java

```diff
@@ -75,6 +76,8 @@ public final String toString() {
             toXContent(builder, EMPTY_PARAMS);
             return builder.string();
         } catch (Exception e) {
+            // So we have a stack trace logged somewhere
+            ESLoggerFactory.getLogger(getClass().getName()).warn("Error building toString", e);;
```

Comment:
yea sorry....... should have been "seems harmless". Yet we may want to change what we do here and whether we need logging or not. And not sneak it in among all these other changes.